src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,31 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <>
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+        <p>
+          {' Posted by  '}
+          <UserInfo user={post.user} />
```

Comment:
in data in this task always exists post.user, but potentially post.user can be undefined,
and UserInfo component can fail in such case.
so better to render UserInfo conditionally

```suggestion
          {post.user && <UserInfo user={post.user} />}
```